pkgs/by-name/li/libeconf/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  meson,
+  ninja,
+  nix-update-script,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libeconf";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "openSUSE";
+    repo = "libeconf";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-xvnyFdsv9sSXeSaKLHkk7DXZAf6XwY6KJfFSSNgnWOI=";
+  };
+
+  nativeBuildInputs = [
+    cmake
```

Comment:
`cmake` should not be needed. Meson supports it for finding dependencies but typically, `pkg-config` will be sufficient. See <https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/meson-cmake.md> for more info.